CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
+* Load the assembly with 'Assembly.LoadFrom' before 'Assembly.Load' when the file path is given (#4196)
+* Add a generic file watcher function in HelpersCommon.psm1 (#4775)
+* Update old links and fix broken links in host-powershell/README.md (#4877)
+* Fix importing remote module using version filters and added tests (#4900)
+* Enable transcription of native commands on non-Windows (#4871)
+* Fix `powershell -version` and help to align with other tools (#4958)
+* Fix incorrect position of a parameter which resulted in the args passed as input instead of as args (#4963)
+* Add a new line to 'CommandNotFoundException' error string (#4934 & #4991)
+
+### General cmdlet updates and fixes
+
+* Added Remove-service to Management module (#4858) (Thanks @joandrsn)
+* Added functionality to set credientials on `Set-Service` command (#4844) (Thanks @joandrsn)
+* `Select-String` changes:
+    * Exclude directories discovered from `-Path` (#4829) (Thanks @iSazonov)
+    * Support more argument completion scenarios by adding `ArgumentCompletionsAttribute` (#4835) (Thanks @iSazonov)
+* Exclude `-Comobject` parameter of `New-Object` on unsupported platforms (#4922) (Thanks @iSazonov)
+* Updated default `ModuleVersion` in `New-ModuleManifest` (#4842) (Thanks @LDSpits)
+* Add Multipart Support to Web Cmdlets (#4782) (Thanks @markekraus)
+* Initialize Web Cmdlets Headers Dictionary Only Once (#4853) (Thanks @markekraus)
+* Change Web Cmdlets UserAgent from `WindowsPowerShell` to `PowerShell` (#4914)
+* Allow * to be used in registry path for `remove-item` (#4866)
+
+### Build and Packaging Improvements
+
+* Make the build output the WiX compilation log if it failed. (#4831) (Thanks @bergmeister)
+* Use a simple file based check in the MSI for the VC++ 2015 redistributables (#4745) (Thanks @bergmeister)
+* Build powershell core using the generic RID 'linux-x64' (#4841)
+* Create generic Linux-x64 packages that are portable to all supported RPM Linux (and more similar for Debian based) distros (#4902 & #4994)
+* Suppress the output of building test tools in Compress-TestContent (#4957)
+* Remove unnecessary error messages from output (#4954)
+* Update travis-ci script so that PRs can fail due to pester tests (#4830)
+* Revert the changes to Pester from merged PR #4184 (#4881)
+* Move release build definition into PowerShell (#4884)
+* Fix credential scan issues (#4927)
+* Add line secret suppression (#4935)
+* Enable security flags in native compiler (#4933)
+* Add VS 2017 solution file for powershell-win-core (#4748)
+
+### Code Cleanup
+
+* Remove remainder of Utility.Activities (Workflow code) (#4880)
+* Remove Microsoft.PowerShell.CoreCLR.AssemblyLoadContext.dll (#4868)
+* Remove unsupported -showwindow switch (#4903)
+* Enable auto EOL on git repo side, fix some character encoding issues (#4912)
+* Autocorrected CRLF to LF (#4943)
+* Change CRLF to LF for files that are already in repository (#4956)
```

Comment:
Combine with #4943.  Just say: Updated EOL for all files to be LF in the repository.

Reply:
Resolved